test-yolo.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+#
+# Usage:
+#
+# Run all tests against PHP 7.3:
+#
+#   ./test-yolo.py ci-drupal-8
+
+
+import os
+import pwd
+import errno
+import pprint
+import re
+import subprocess
+import random
+import signal
+import sys
+import time
+import json
+
+import requests
+import unittest
+import shlex
+
+# pass in site name as an argument to specify a Drupal 7 or Drupal 8 site
+
+siteName = sys.argv[1]
+del sys.argv[1]
+
+commandPrefix = "terminus drush "
+
+#drupal7Site = "ci-drupal-7"
+#drupal8Site = "ci-drupal-8"
+
+class EnvCacheClearTestCase(unittest.TestCase):
+    def fixtureCommand(self):
+        return "terminus env:clear-cache " + siteName + ".dev"
+
+    def getFixtureResult(self):
```

Comment:
Again this is best not called "...Fixture...". Only two lines, but I wonder if refactoring would be of value here. rpmbuild-php has this same issue.